.github/workflows/gating.yaml

```diff
@@ -134,6 +138,12 @@ jobs:
             --output ./cachi2-output \
             --package '{"path": ".", "type": "gomod"}'
 
+      - name: Run integration tests on built image
+        env:
+          CACHI2_IMAGE: localhost/cachi2:${{ github.sha }}
+        run: |
+           pytest tests/integration
```

Comment:
Should run `tox -e integration` instead? And we wouldn't have to manually install the requirements-test.txt that way

Reply:
Depends on you guys, what you prefer. Even tox needs to be pre-installed in github CI run.
* https://github.com/containerbuildsystem/cachi2/actions/runs/3685151199/jobs/6235758076 
* https://github.com/containerbuildsystem/cachi2/blob/main/.github/workflows/gating.yaml#L35
Thoughts?

Reply:
Done